math/genvector/inc/Math/GenVector/Cartesian2D.h

```diff
@@ -89,8 +89,9 @@ public :
    Scalar X()     const { return fX;}
    Scalar Y()     const { return fY;}
    Scalar Mag2()  const { return fX*fX + fY*fY; }
-   Scalar R()     const { return std::sqrt( Mag2());}
-   Scalar Phi()   const { return (fX==0 && fY==0) ? 0.0 : atan2(fY,fX);}
+   Scalar R()     const { using namespace std; return sqrt( Mag2()); }
```

Comment:
I am sure I ran into some issue that lead to to doing this... Also note the templation is in principle  there for other type than Vc. This change in principle supports types that do not extend the math functions under std::, as well as those that do.